addons/load-extensions/addon.json

```diff
@@ -6,6 +6,11 @@
       "type": "warning",
       "text": "When the video sensing option is enabled, the webcam will turn on every time the editor is opened.",
       "id": "webcam"
+    },
+    {
+      "type": "notice",
+      "text": "If an extension monitor, for example, the tempo is shown on the stage, that extension will stay loaded in that project even after hiding all monitors and disabling the addon.",
```

Comment:
Very sad

Reply:
I didn't know that.

Reply:
The combination of these two warnings is very scary.

Reply:
> The combination of these two warnings is very scary.

That's what I was worried about.

Reply:
#3930 would help here.